baseplate/clients/requests.py

```diff
@@ -26,6 +27,9 @@
 from baseplate.lib.prometheus_metrics import getHTTPSuccessLabel
 
 
+RequestsInstrumentor().instrument()
```

Comment:
Honestly I don't know. I think it would make testing much more complex since you'd need to basically load the entire app to test anything.
It's also how otel says you should be using these auto-instrumenter.

Reply:
I agree with Chris that this smells a little funny. I don't see anything in their API docs that suggests that this needs to be instrumented the top level like this. Can you share the link to the instructions/example you are following?

Reply:
Can you check the commit I just pushed? Is that closer to what y'all had in mind? (we change names etc. I just wanted to get smth out there for you to be able to look at)